config.js

```
@@ -0,0 +1,16 @@
+const dotenv = require('dotenv');
+dotenv.config();
+
+module.exports = {
+  SERVER_ADDRESS: process.env.SERVER_ADDRESS || '127.0.0.1',
+  SERVER_PORT: parseInt(process.env.SERVER_PORT || 8765),
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)